# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/util-linux/disable_cramfs_tools.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./disk-utils/Makefile.orig	2004-12-30 17:55:07.774194368 +0100
+++ ./disk-utils/Makefile	2004-12-30 17:56:04.443579312 +0100
@@ -20,7 +20,7 @@
 
 ETC=	fdprm
 
-MAYBE=  setfdprm raw fsck.cramfs mkfs.cramfs
+MAYBE=  setfdprm raw
 
 ifneq "$(HAVE_FDUTILS)" "yes"
 USRBIN:=$(USRBIN) setfdprm
@@ -32,26 +32,12 @@
 MAN8:=$(MAN8) raw.8
 endif
 
-ifeq "$(HAVE_ZLIB)" "yes"
-SBIN:=$(SBIN) fsck.cramfs mkfs.cramfs
-endif
-
 ifeq "$(HAVE_UUID)" "yes"
 MKSWAP_LIBS=-luuid
 endif
 
 all: $(SBIN) $(USRBIN)
 
-fsck.cramfs: fsck.cramfs.o
-	$(CC) $(LDFLAGS) -o fsck.cramfs fsck.cramfs.o -lz
-
-mkfs.cramfs: mkfs.cramfs.o $(LIB)/md5.o
-	$(CC) $(LDFLAGS) -o mkfs.cramfs mkfs.cramfs.o $(LIB)/md5.o -lz
-
-mkfs.cramfs.o: $(LIB)/md5.h
-
-fsck.cramfs.o mkfs.cramfs.o: cramfs.h
-
 fsck.minix.o mkfs.minix.o: bitops.h minix.h
 
 mkswap: mkswap.o $(LIB)/xstrncpy.o
